src/lib/config/ModelConfig.cc

```diff
@@ -886,6 +886,18 @@ void ModelConfig::postValidation() {
   // Record any unlinked port names
   for (const auto& prt : portnames)
     invalid_ << "\t- " << prt << " has no associated reservation station\n";
+
+  // Check for valid LSQ-L1-Interface Load and Store Bandwidth
+  if (configTree_["Core"]["Vector-Length"].as<uint16_t>() / 8 >
+      configTree_["LSQ-L1-Interface"]["Load-Bandwidth"].as<uint16_t>()) {
+    invalid_ << "\t- Vector-Length (bits) must be greater than Load-Bandwidth "
+                "(bytes)\n";
+  } else if (configTree_["Core"]["Vector-Length"].as<uint16_t>() / 8 >
```

Comment:
Should this be a separate if rather than else if? Would allow for both issues to printed to the user on the first attempt. No concerns for performance as this only runs once at the start of the program

Reply:
Good idea. Implemented and works as expected